### PR TITLE
added sigmoid, relu, init_params

### PR DESCRIPTION
**Pull request by elisim**

### CODE REVIEW

File: assignment1/fc_net.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        self.num_layers = 1 + len(layer_dims)
+        
+        
+def initialize_parameters(layer_dims):
+    """
+    input: 
+        an array of the dimensions of each layer in the network (layer 0 is the size of the flattened input, layer L is the output sigmoid)
+    output: 
+        a dictionary containing the initialized W and b parameters of each layer (W1...WL, b1...bL).
+    Hint:
```
Comment (galbraun) [inline assignment1/fc_net.py]: Please remove the hint from the comment, also I think we should use python docstring as you did for the class

File: assignment1/layers.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        A – the activations of the previous layer
+        W – the weight matrix of the current layer (of shape [size of current layer, size of previous layer])
+        B – the bias vector of the current layer (of shape [size of current layer, 1])
+
+    Output:
+        Z – the linear component of the activation function (i.e., the value before applying the non-linear function)
+        linear_cache – a dictionary containing A, W, b (stored for making the backpropagation easier to compute)
+    """ 
+    n_activations = A.shape[0]
+    Z = np.dot(A.reshape(n_activations,-1), W) + b
```
Comment (galbraun) [inline assignment1/layers.py]: why do you use this reshape? I think A should be a matrix - for a batch of samples no?

File: assignment1/layers.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+def relu(Z):
+    """
+    Input:
+        Z – the linear component of the activation function
+    Output:
+        A – the activations of the layer
+        activation_cache – returns Z, which will be useful for the backpropagation
+    """
+    A = Z * (Z >= 0)
```
Comment (galbraun) [inline assignment1/layers.py]: Nice trick!

File: assignment1/fc_net.py (hunk 1 above)
Comment (elisim) [reply assignment1/fc_net.py]: I just copied it from the assignment description. hint removed

File: assignment1/layers.py (hunk 2 above)
Comment (elisim) [reply assignment1/layers.py]: Okay, so what is A.shape? 
I got my inspiration from my cs231n code:
https://github.com/elisim/cs231n/blob/7e0e8ec8cb675b1a52398bcb59576d995e8f8ce8/assignment2/cs231n/layers.py#L5

File: assignment1/layers.py (hunk 3 above)
Comment (elisim) [reply assignment1/layers.py]: %kiss%

**Pull request merged**